test/python/pulse/test_schedule.py

```diff
@@ -121,7 +121,7 @@ def test_can_create_valid_schedule(self):
         sched = sched.insert(90, Acquire(10,
                                          self.config.acquire(0),
                                          MemorySlot(0),
-                                         RegisterSlot(0)))  # TODO: this shouldn't raise a warning
+                                         reg_slot=RegisterSlot(0)))
```

Comment:
I think you missed the point, this is testing previous behavior. It shouldn't raise a warning not because it needs to be updated but because users who have this shouldn't have to make changes. 
```suggestion
                                         RegisterSlot(0)))
```
I'm not sure it's possible to get what we want here. We might have to wait until the deprecation period is over and the warnings are removed.

Reply:
Fixed in bf5c3388a